gramps/gui/viewmanager.py

```diff
@@ -580,10 +580,47 @@ def no_del_event(self, *obj):
         hits 'x' multiple times. """
         return True
 
+    def checkdisk(self, dbfolder): # TODO check this function
+        """
+        Check disk space
+        """
+        if win():                               # for windows:
```

Comment:
What should we do for Windows?

Reply:
Gramps work in Windows too. it's draft for Windows. I check this code in my Linux system, but I have not Windows.